src/components/calcite-value-list/calcite-value-list.tsx

```diff
@@ -146,16 +148,18 @@ export class CalciteValueList {
   }
 
   setUpDragAndDrop(): void {
-    const sortGroups = [
-      this.el,
-      ...Array.from(this.el.querySelectorAll("calcite-value-list-group"))
-    ];
+    const sortGroups = [this.el];
     sortGroups.forEach((sortGroup: HTMLElement) => {
       this.sortables.push(
         Sortable.create(sortGroup, {
           group: this.guid,
           handle: `.${CSS.handle}`,
-          draggable: "calcite-value-list-item"
+          draggable: "calcite-value-list-item",
+          onUpdate: () => {
+            this.items = Array.from(this.el.querySelectorAll("calcite-value-list-item"));
+            const values = this.items.map((item) => item.value);
+            this.calciteValueListOrderChange.emit(values);
```

Comment:
Is this payload needed? Users can query for the items if needed.

Reply:
@dhrumil83 can you weigh in on if you would need this or find this useful?

Reply:
I just need what moved and where via drag and drop so I can update the array. 
If I get the all values, I can check what changed myself and update.

Reply:
ok sounds useful. keeping.